react/src/theme/components/ProgressIndicator.ts

```diff
@@ -0,0 +1,40 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+import { anatomy } from '@chakra-ui/theme-tools'
+
+const parts = anatomy('attachment').parts(
+  'circleIndicator',
+  'activeIndicator',
+  'container',
+)
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(parts.keys)
+
+const baseStyle = definePartsStyle(({ isActiveIndicator, colorMode }) => ({
+  circleIndicator: {
+    width: '0.75rem',
+    height: '0.75rem',
+    padding: '0.125rem',
+    borderRadius: 'full',
+    backgroundColor: 'interaction.support.unselected',
+    mr: isActiveIndicator ? '1.25rem' : '0.25rem',
```

Comment:
for "correctness", should we use the `_active` prop to update the margin?

Reply:
yessir